src/components/Reports/ExpectedMonthlyTotalReport/Header/ExpectedMonthlyTotalReportHeader.tsx

```diff
@@ -0,0 +1,64 @@
+import React from 'react';
+import { Box, Typography, styled, Divider } from '@material-ui/core';
+import { useTranslation } from 'react-i18next';
+
+interface Props {
+  empty: boolean;
+}
+
+const HeaderWrapper = styled(Box)(({}) => ({
+  display: 'flex',
+  flexDirection: 'row',
+  justifyContent: 'space-between',
+  padding: 24,
+}));
+
+const ProgressBarWrapper = styled(Box)(({}) => ({
+  width: 450,
+  display: 'flex',
+  flexDirection: 'row',
+  height: 35,
+  border: '2px solid inherit',
+  borderRadius: 30,
+  overflow: 'hidden',
+}));
+
+const ProgressBarSection = styled(Box)(({}) => ({
+  height: '100%',
+  textAlign: 'center',
+  color: 'white',
+  flexGrow: 1,
+}));
+
+export const ExpectedMonthlyTotalReportHeader: React.FC<Props> = ({
+  empty,
+}) => {
+  const { t } = useTranslation();
+  return (
+    <Box>
+      <HeaderWrapper>
+        <Typography variant="h4">{t('Expected Monthly Total')}</Typography>
+        {empty === false ? (
+          <ProgressBarWrapper>
+            <ProgressBarSection
+              style={{ backgroundColor: '#f9b625', width: '36%' }}
```

Comment:
My guess is that these hex values have entries in the theme. We should try to use those if possible.